routersploit/modules/exploits/dlink/dcs_930l_932l_auth_bypass.py

```diff
@@ -0,0 +1,121 @@
+import re
+from routersploit import (
+    exploits,
+    print_error,
+    print_table,
+    print_success,
+    http_request,
+    mute,
+    validators
+)
+
+
+class Exploit(exploits.Exploit):
+    """
+    D-Link DCS web cameras allow unauthenticated attackers to obtain the
+    configuration of the device remotely. A copy of the device configuration can be
+    obtained by accessing unprocteted URL.
+    """
+    __info__ = {
+        'name': 'D-Link DCS Cameras Authentication Bypass',
+        'description': 'D-Link DCS web cameras allow unauthenticated attackers to obtain the configuration of the device remotely.'
+                       'A copy of the device configuration can be obtained by accessing unprocteted URL.',
+        'authors': [
+            'Roberto Paleari',  # vulnerability discovery
```

Comment:
Feel free to add your name as a routersploit module author "# routersploit module" or ignore it if your are Roberto :)